readme.md

```diff
@@ -94,17 +97,141 @@ Visit <https://indiafightscorona.giveindia.org> and make a small donation to hel
+If you want to generate multiple stats, such as top languages, wakatime, etc, you can use the following code.
+
+```yml
+name: GitHub Readme Stats
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: 0 * * * *
+
+jobs:
+  stats:
+    permissions: write-all
+    runs-on: ubuntu-latest
+    name: Stats Card
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Generate Stats
+        uses: Zo-Bro-23/grs-action@v1
+        id: generate
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          options: 'username=${{ github.repository_owner }}'
+      - uses: actions/upload-artifact@master
+        with:
+          name: stats
+          path: grs
+  top-langs:
+    permissions: write-all
+    runs-on: ubuntu-latest
+    name: Top Languages Card
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Generate Top Languages
+        uses: Zo-Bro-23/grs-action@v1
+        id: generate
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          options: 'username=${{ github.repository_owner }}'
+          card: 'top-langs'
+      - uses: actions/upload-artifact@master
+        with:
+          name: top-langs
+          path: grs
+  push:
+    permissions: write-all
+    runs-on: ubuntu-latest
+    name: Push
+    needs: [stats, top-langs]
+    steps:
+      - uses: actions/download-artifact@master
+        with:
+          name: stats
+          path: grs
+      - uses: actions/download-artifact@master
+        with:
+          name: top-langs
+          path: grs
+      - name: Push
+        uses: crazy-max/ghaction-github-pages@v3
+        with:
+          target_branch: grs
+          build_dir: grs
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+```
+
+You can now include the stats in your README by using the following markdown code (replace `USERNAME` with your GitHub username).
+
+```md
+[![Your GitHub stats](https://github.com/USERNAME/USERNAME/raw/grs/stats.svg)](https://github.com/anuraghazra/github-readme-stats)
+```
+
+You can also enable GitHub Pages on the `grs` branch (root) and use that URL (replace `GH_PAGES_URL` with your GitHub Pages URL).
+
+```md
+[![Your GitHub stats](https://GH_PAGES_URL/stats.svg)](https://github.com/anuraghazra/github-readme-stats)
+```
 
-Copy-paste this into your markdown content, and that is it. Simple!
+The GitHub Action accepts the following inputs:
+- `token` - `REQUIRED` - Your GitHub Personal Access Token (PAT), or any other token that GitHub accepts.
+- `options` - `REQUIRED` - Any customization options. More below. (This is the same as the query string you would pass to the API if using the community deployment)
+- `card` - The type of card to generate. `repo`, `top-langs`, or `wakatime` (leave empty for stats card).
+- `path` - Output path for SVG file. Relative path; include filename with `.svg`.
+- `fetch_multipage` - Experimental feature that gives more accurate stats if you have >100 repos.
```

Comment:
As discussed on discord, this can be removed and enabled by default.

Reply:
Yes, I will implement that.

Reply:
Cool thanks! People can still disable it by changing the environment variable. This variable has to be documented in the GRS readme. We can do that later.